helix-term/src/commands.rs

```diff
@@ -1031,19 +1031,24 @@ fn goto_file_impl(cx: &mut Context, action: Action) {
         .collect();
     let primary = selections.primary();
     if selections.len() == 1 && primary.to() - primary.from() == 1 {
```

Comment:
Unrelated to this change but could this be `primary.len() == 1`? I don't think we had `Range::len` when this block was originally written.

It's the same implementation but it's a bit clearer - `to - from` mistakenly makes me think that we'll run into an issue if the range is reversed (which isn't true actually) 😄

Reply:
> Unrelated to this change but could this be `primary.len() == 1`? I don't think we had `Range::len` when this block was originally written.
> 
> It's the same implementation but it's a bit clearer - `to - from` mistakenly makes me think that we'll run into an issue if the range is reversed (which isn't true actually) smile

I have replaced it with `primary.len() == 1` and it works fine